k-distribution/tests/regression-new/map-symbolic-tests-haskell/assignment-29-spec.k.out

```diff
@@ -1 +1,23 @@
-#Top
+  #Not ( {
+    X
+  #Equals
+    Y
+  } )
+#And
+  <k>
+    assignmentResult ( MAP
+    X:MyId |-> BAL_TO:Int
+    Y:MyId |-> BAL_FROM +Int -30 ) ~> .
```

Comment:
Should this be `-Int 30` instead?

Reply:
I think there is a simplification rule for this, but maybe I'm thinking of EVM?

Reply:
No, the arithmetic simplification rules are in `domains.k` now. But it looks like this particular result comes from the application of [this](https://github.com/kframework/k/blame/master/k-distribution/include/kframework/builtin/domains.md#L1028) rule.